From d6e05e7a26fc512995f405b8fad8df07f1e0326f Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Sat, 17 Sep 2016 17:49:35 -0700
Subject: [PATCH] joyent/node-triton#131 update 'make cutarelease' and 'make
 versioncheck' to avoid commits for cutting a release (fix 'npm publish'
 command usage)

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6951165..b74b5b4 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ cutarelease: versioncheck
 	    date=$(shell date -u "+%Y-%m-%d") && \
 	    git tag -a "$$ver" -m "version $$ver ($$date)" && \
 	    git push --tags origin && \
-	    npm publish $$name
+	    npm publish
 
 .PHONY: git-hooks
 git-hooks:
-- 
2.21.0

